SetupDataPkg/ConfProfileMgrDxe/ConfProfileMgrDxe.c

```diff
@@ -105,7 +105,18 @@ ValidateActiveProfile (
       }
     }
 
-    if (VariableInvalid || (0 != CompareMem (Data, VarList[i].Data, VarList[i].DataSize))) {
+    if (!VariableInvalid) {
+      if (!IsSystemInManufacturingMode ()) {
+        DEBUG ((DEBUG_INFO, "%a System not in MFG Mode, validating profile matches variable storage\n", __FUNCTION__));
+        if (0 != CompareMem (Data, VarList[i].Data, VarList[i].DataSize)) {
+          VariableInvalid = TRUE;
+        }
+      } else {
+        DEBUG ((DEBUG_INFO, "%a System in MFG Mode, not validating profile matches variable storage\n", __FUNCTION__));

```

Comment:
Can we change this print to read something like: `System in MFG Mode, not validating profile data matches variable storage data, only validating size and attributes.`